### ISSUE TITLE
Docker-compose needs to be leaner

### ISSUE THREAD

**Issue by ucyo** [climate-v/nc2zarr-webapp#6]
Using `docker-compose.yaml` helps streamline the development and production environments.    A repository making heavy use of compose usually has multiple configuration files. Most common are three: 

- `docker-compose.yaml` with all basic configuration 
- `docker-compose.override.yaml` with configuration details for development environments
- and `docker-compose.prod.yaml` with configuration for production environment.

Starting the local instance in dev mode use `docker compose up`. The `override` configuration is automatically read and run. If you want to run the production environment run the following command:

``` shell
docker compose -f docker-compose.yaml -f docker-compose.prod.yaml up
```

We currently don't have a production environment, but we still should stick to the best practice described above. Currently the base `docker-compose.yaml` file as too many specific configurations for the development environment. They must be moved to `docker-compose.override.yaml`. 

For example: 
- rebrow and rq-dashboard can be moved to override completely.
- ports are hardcoded in base and should be read via environment variables
- all port exposes should be moved to dev environment

Reference:
- https://docs.docker.com/compose/extends/#multiple-compose-files

**Comment by b-pos465** [comment]
Thanks for the hint! 

I just found out that there is another mechanism called `profiles`. This could also be intersting to provide an environment for development: https://stackoverflow.com/questions/30233105/docker-compose-up-for-only-certain-containers

**Comment by b-pos465** [comment]
Fixed in 4310f4ac5d3f1a88c487146848395df49fffbf0d.
